Frontend/src/components/Sidebar.tsx

```diff
@@ -0,0 +1,50 @@
+import { Box, Divider, Flex, Image } from "@chakra-ui/react"
+import React from "react"
+import { Link } from "react-router-dom"
+import mycollections from "../assets/mycollections.svg"
+import vote from "../assets/vote.svg"
+
+interface Props {
+  children: React.ReactNode
+}
+
+export default function Sidebar({ children }: Props) {
+  return (
+    <Flex w="full" h="calc(100% - 60px)" px={8} pt={8}>
+      <Flex w={60} h="full" flexDir="column" gap={6}>
+        <Link to="/voting-overview">
+          <Flex alignItems="center" gap={4}>
+            <Box w={6}>
+              <Image src={vote} alt="logo" />
+            </Box>
+            Voting Overview
```

Comment:
I think we should comment this until the page is ready

Reply:
Yeah thanks, forgot it.